python/interpret-core/interpret/utils/_explanation.py

```diff
@@ -137,7 +137,20 @@ def gen_local_selector(data_dicts, round=3, is_classification=True):
         record["Predicted"] = perf_dict["predicted"]
         record["Actual"] = perf_dict["actual"]
 
-        record["Resid"] = record["AcScore"] - record["PrScore"]
+        # Handle case where AcScore might be string (from typify_classification of float labels)
+        # while PrScore is always numeric (from model predictions)
+        ac_score = record["AcScore"]
+        pr_score = record["PrScore"]
+        
+        # Convert AcScore to float if it's a string for arithmetic operations
+        if isinstance(ac_score, (str, np.str_)):
+            try:
+                ac_score = float(ac_score)
+            except (ValueError, TypeError):
+                # If conversion fails, use NaN for residual calculation
+                ac_score = np.nan
+        
+        record["Resid"] = ac_score - pr_score
```

Comment:
This section should be undone since AcScore and PrScore should both be floats even for classification